.github/CODEOWNERS

```diff
@@ -74,7 +74,7 @@
 /avm/res/managed-identity/user-assigned-identity/ @Azure/avm-res-managedidentity-userassignedidentity-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/managed-services/registration-definition/ @Azure/avm-res-managedservices-registrationdefinition-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/management/management-group/ @Azure/avm-res-management-managementgroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/net-app/net-app-account/ @Azure/avm-res-netapp-netappaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/net-app/net-app-account/ @Azure/avm-res-netapp-netappaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Please ref to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
@Azure/avm-res-netapp-netappaccount-module-owners-bicep
@Azure/avm-res-netapp-netappaccount-module-contributors-bicep